Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG PHP_VERSION=8.1.8
-
-FROM php:${PHP_VERSION}-fpm-alpine
+FROM php:8.1.8-fpm-alpine
```

Comment:
Lets back to ARG version. Or without this change dependabot cannot work properly?